staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -394,6 +395,19 @@ var (
 			},
 		},
 	)
+	// the following flow schema exempts probes
+	SuggestedFlowSchemaProbes = newFlowSchema(
+		"probes", "exempt", 2,
+		flowcontrol.FlowDistinguisherMethodByUserType,
```

Comment:
The choice of flow distinguisher method is not obvious.  Since these requests are exempt and thus not queued, there is no wrong answer from a functional point of view.  Why not go with the simplest choice, the one that makes the flow distinguisher always be the empty string?  Specifying something more specific suggests there is a reason, which would be misleading to readers if there is not a reason.  Is there thinking that this will produce better metrics or debug output or something? If there is a positive reason, it would be best to comment it, since it is not obvious.